pyomo/contrib/mindtpy/tests/test_mindtpy_lp_nlp.py

```diff
@@ -57,6 +57,20 @@ def test_LPNLP(self):
                 self.assertAlmostEqual(
                     value(model.objective.expr), model.optimal_value, places=1)
 
+    def test_LPNLP_GUROBI(self):
```

Comment:
I think it would make more sense just to have all the tests run with an extra argument being the mip solver, cplex_persistent or gurobi_persistent